src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -2888,12 +3647,23 @@ subroutine internal_tides_init(Time, G, GV, US, param_file, diag, CS)
 
   ! residual
   allocate(CS%residual(isd:ied,jsd:jed), source=0.0)
-  do j=G%jsc,G%jec ; do i=G%isc,G%iec
-    if (CS%refl_pref_logical(i,j)) then
-      CS%residual(i,j) = 1. - CS%refl_pref(i,j) - CS%trans(i,j)
-    endif
-  enddo ; enddo
-  call pass_var(CS%residual, G%domain)
+  if (CS%apply_residual_drag) then
+    do j=G%jsc,G%jec ; do i=G%isc,G%iec
+      if (CS%refl_pref_logical(i,j)) then
+        CS%residual(i,j) = 1. - CS%refl_pref(i,j) - CS%trans(i,j)
```

Comment:
Should we add parentheses here to specify the order of this 3-element sum?

Reply:
done